src/lib/relay/routes.tsx

```diff
@@ -54,15 +54,20 @@ class Gene extends Relay.Route {
 
 class WorksForYou extends Relay.Route {
   static queries = {
-    me: (component, params) => Relay.QL`
+    viewer: (component, params) => Relay.QL`
       query {
-        me {
-          ${component.getFragment("me", params)}
+        viewer {
+          ${(component.getFragment("viewer", params))}
```

Comment:
I could do that, but then the `artworks` would have to be `me.selectedArtist` or something which isn't entirely true since the `artist` doesn't reside under `me` in graphql.